data/networks.ts

```diff
@@ -119,6 +150,74 @@ const publicChains: ZkSyncNetwork[] = [
     l1Network: l1Networks.sepolia,
     hidden: true,
   },
+  {
```

Comment:
Why did we add gateway networks to the era portal dropdown? I think the idea was to keep era and gateway in a separate dropdowns using something like group id.

Reply:
Checkout out [the explorer](https://explorer.zksync.io/), it's the same instance, but different dropdown is shown depending on selected network.